__tests__/generate.test.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+const generate = require('../db/seeding/generateCore.js');
+
+const testFilePath = path.join(__dirname, 'test.json');
+
+beforeAll(done => generate('../../__tests__/test.json', 10, done));
+
+test('the file is created', (done) => {
```

Comment:
I'm not entirely sure what information jest returns if any one of these expect statements fail, but would it maybe be helpful to break out some of these tests for specific properties so that if one does fail it would be easier to tell which property is causing the failure?

Reply:
Per your suggestion, I broke out the expect statements into separate tests.